doc/content/syntax/SpatioTemporalPaths/index.md

```diff
@@ -0,0 +1,32 @@
+# SpatioTemporalPath System
```

Comment:
More of a general comment/question - have you thought about (or do you see) this system being applied to some of our current laser melting and laser welding examples? I wonder if there's some quality of life updates to be made in some of these older examples after this goes in....